### DISCUSSION TITLE
能否添加 scheme  URL 功能

### DISCUSSION THREAD

**Discussion by kiwiflydream** [he3-app/he3-feedback#124] [Ideas]
比如通过 "he3://v1/text?string_escapes" 可以快速打开字符串转义界面，这样就可以跟很多效率工具集成了，大大提高了便捷性

**Comment by lyzhang1999** [comment]
非常好的建议！我已经建立了一个新的 issues 来跟踪这个问题：https://github.com/he3-app/he3-feedback/issues/125

**Comment by lyzhang1999** [comment]
PS：目前是可以通过 scheme URL 打开任意一款工具的，但还无法将参数传递到工具内，你可以在该页面找到所有可用的工具以及用于打开工具的链接：https://portal.he3app.com/tools?page=1

**Comment by zhenweiwang1990** [comment]
我们正在评估如何通过 he3:// 协议链接传递对应 input 内容给工具，这样以来，就可以实现你想要的效果了。再次感谢建议！♥️
